diff --git a/config/autoload/dependencies.global.php b/config/autoload/dependencies.global.php
index 163a769e..8670abad 100644
--- a/config/autoload/dependencies.global.php
+++ b/config/autoload/dependencies.global.php
@@ -34,7 +34,7 @@ return [
         ],
 
         'lazy_services' => [
-            'proxies_target_dir' => 'data/proxies',
+            'proxies_target_dir' => '/var/lib/shlink/proxies',
             'proxies_namespace' => 'ShlinkProxy',
             'write_proxy_files' => EnvVars::isProdEnv(),
         ],
diff --git a/config/autoload/entity-manager.global.php b/config/autoload/entity-manager.global.php
index 4f547f54..d560a5e9 100644
--- a/config/autoload/entity-manager.global.php
+++ b/config/autoload/entity-manager.global.php
@@ -50,7 +50,7 @@ return (static function (): array {
     $connection = match ($driver) {
         null, 'sqlite' => [
             'driver' => 'pdo_sqlite',
-            'path' => 'data/database.sqlite',
+            'path' => '/var/lib/shlink/database.sqlite',
         ],
         default => [
             'driver' => $doctrineDriver,
@@ -68,7 +68,7 @@ return (static function (): array {
     return [
         'entity_manager' => [
             'orm' => [
-                'proxies_dir' => 'data/proxies',
+                'proxies_dir' => '/var/lib/shlink/proxies',
                 'load_mappings_using_functional_style' => true,
                 'default_repository_classname' => EntitySpecificationRepository::class,
                 'listeners' => [
diff --git a/config/autoload/locks.global.php b/config/autoload/locks.global.php
index d94ba5bd..c2ece472 100644
--- a/config/autoload/locks.global.php
+++ b/config/autoload/locks.global.php
@@ -13,7 +13,7 @@ use const Shlinkio\Shlink\LOCAL_LOCK_FACTORY;
 
 return [
     'locks' => [
-        'locks_dir' => __DIR__ . '/../../data/locks',
+        'locks_dir' => '/tmp/shlink-data/locks',
     ],
 
     'dependencies' => [
diff --git a/config/autoload/router.global.php b/config/autoload/router.global.php
index a749407c..dab7c4dd 100644
--- a/config/autoload/router.global.php
+++ b/config/autoload/router.global.php
@@ -13,7 +13,7 @@ return [
             // Disabling config cache for cli, ensures it's never used for RoadRunner, and also that console
             // commands don't generate a cache file that's then used by php-fpm web executions
             FastRouteRouter::CONFIG_CACHE_ENABLED => EnvVars::isProdEnv() && PHP_SAPI !== 'cli',
-            FastRouteRouter::CONFIG_CACHE_FILE => 'data/cache/fastroute_cached_routes.php',
+            FastRouteRouter::CONFIG_CACHE_FILE => '/var/lib/shlink/cache/fastroute_cached_routes.php',
         ],
     ],
 ];
diff --git a/module/Core/src/ObjectMapper/MapperBuilderWithCacheDelegatorFactory.php b/module/Core/src/ObjectMapper/MapperBuilderWithCacheDelegatorFactory.php
index 1db93c08..9f88847e 100644
--- a/module/Core/src/ObjectMapper/MapperBuilderWithCacheDelegatorFactory.php
+++ b/module/Core/src/ObjectMapper/MapperBuilderWithCacheDelegatorFactory.php
@@ -20,7 +20,7 @@ class MapperBuilderWithCacheDelegatorFactory
         /** @var MapperBuilder $mapperBuilder */
         $mapperBuilder = $callback();
 
-        $cache = new FileSystemCache('data/cache/valinor');
+        $cache = new FileSystemCache('/var/lib/shlink/cache/valinor');
         if (!EnvVars::isProdEnv()) {
             $cache = new FileWatchingCache($cache);
         }
